lib/utils/rename.js

```diff
@@ -1,16 +1,182 @@
 'use strict'
 var fs = require('graceful-fs')
+var path = require('path')
+var rimraf = require('rimraf')
```

Comment:
Great suggestion! I didn't realize what gentlyRm was doing, but it seems reasonable :)